src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -346,6 +346,9 @@ public function preprocessArticles($linkedArticles, $settings)
                 // Reduce text to max allowed characters
                 $ellipsis = true;
                 $maxChars = 200;
+                $referer = parse_url($_SERVER['HTTP_REFERER'], PHP_URL_HOST);
```

Comment:
On second thought, it might be better to get this info from the `$this->request` property no?

Reply:
Should be able with something like: `$this->request->server->get('HTTP_REFERER');` I think. But you should test it, I'm writing this from the top of my head.

Reply:
Jup it works 👍